nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -522,6 +218,21 @@ You need to fire emag the fighter's IFF board. This makes it list as "ENEMY" on
 		var/datum/gas_mixture/removed = cabin_air.remove(5)
 		qdel(removed)
 
+	var/obj/item/fighter_component/secondary/utility/passenger_compartment_module/pc = get_part(/obj/item/fighter_component/secondary/utility/passenger_compartment_module)
+	if(pc?.burntout)
+		var/datum/gas_mixture/removed = cabin_air.remove(5)

```

Comment:
I look forward to our pilots continuing to manage to suffocate themselves inside a perfectly good capital ship

Reply:
I should resolve that to check for exterior atmos at some point.